frontend/src/components/contentDisplay/DesignOptions.ts

```diff
@@ -0,0 +1,25 @@
+export const colors = [
+  'black',
+  'red',
+  'green',
+  'blue',
+  'orange',
+  'yellow',
+  'purple',
+  'white',
+];
+export const defaultTextColor = 'black';
+export const sizes = ['10px', '14px', '18px', '32px', '64px'];
+export const defaultTextSize = '14px';
+export const fontFamilies = [
```

Comment:
I'd suggest when we go over the fonts to make them cross-platform, we also check these again

Reply:
https://www.w3schools.com/cssref/css_websafe_fonts.php